.github/workflows/conformance-clustermesh.yaml

```diff
@@ -345,6 +407,16 @@ jobs:
             --collect-sysdump-on-failure \
             --sysdump-output-filename 'cilium-sysdump-${{ matrix.name }}-<ts>'" \
 
+          if [ "${{ matrix.multipool-ipam }}" != "disabled" ]; then
+            CONNECTIVITY_TEST_DEFAULTS="$CONNECTIVITY_TEST_DEFAULTS \
+              --external-cidr 8.0.0.0/8 --external-ip 8.8.4.4 --external-other-ip 8.8.8.8 \
```

Comment:
I assume this is a copy/paste leftover.

Reply:
They are probably not needed, let's try a run without them :+1: